### ISSUE TITLE
Paheal compatibility continues to be shaky/inaccessible

### ISSUE THREAD

**Issue by DarkPulse** [NO-ob/LoliSnatcher_Droid#256]
Some months back I opened issue #241 in response to Paheal moving their image servers. After the most recent update, Lolisnatcher worked, but there was still some images that it wound up being confused by.

However, as of today, it's not even turning up results at all. Something has changed in its configuration, and even trying to test for the booru with the previous settings fails (it claims a 404); however, visiting the site naturally still works and proves the site is up and running.

So in short, it's busted once more somehow.

**Comment by NANI-SORE** [comment]
They removed the API we used to fetch data (hence the 404), will have to scrape html now, that will be in the next update. I suspect their managemnt changed and now they are doing some big changes on the backend

**Comment by NANI-SORE** [comment]
Fixed in 2.4.1
